src/controller.cpp

```diff
@@ -5,48 +5,41 @@
 namespace mppi {
 
 void Controller::configure(
-  const rclcpp_lifecycle::LifecycleNode::WeakPtr & parent,
-  std::string node_name, const std::shared_ptr<tf2_ros::Buffer> & tf,
+  const rclcpp_lifecycle::LifecycleNode::WeakPtr & parent, std::string node_name,
+  const std::shared_ptr<tf2_ros::Buffer> & tf,
   const std::shared_ptr<nav2_costmap_2d::Costmap2DROS> & costmap_ros)
 {
-  // Set inputs
-  parent_ = parent;
   auto node = parent.lock();
-  logger_ = node->get_logger();
 
-  costmap_ros_ = costmap_ros;
-  tf_buffer_ = tf;
-  node_name_ = node_name;
+  { // Set inputs
```

Comment:
Why are these being put into scope brackets? I don't see those having any value in this function -- there's no mutex being locked or something needing scope differentiation.

Reply:
Seems to me that It helps to emphasise that this part of code is logic unit and not affects rest of the func and seems more readable. Though, could be rolled back easily, i think it's more about stylistic

Reply:
OK, I don't mind. There are a few style things that will have to be linted out before it gets moved to the Nav2 repo to match the style guide, but its fine to keep them here. We have auto-linters run in our CI over all of the code so we have to comply with it in Nav2 or else CI will yell at me.